src/components/Cards/subComponents/Card.js

```diff
@@ -64,6 +64,31 @@ const FlashCard = ({ card, classes }) => {
 
 export default withStyles(styles)(FlashCard); 
 
+/* -------------------------------------------------------------------------- */
+/*                             proposed structure                             */
+/*
+    * 4/10/2021 - I'm still learning. File structure has a huge impact on a project's 
+    * future. This structure may change as I learn more and improve my practices.      
+    components
+        | Card 📁
+            | CardStyles.js
+            | index.js -> Card component itself 
+    
+    ? Should the Cards component be in this directory?? Or should it be its own modular directory? 
+    ! HARRISON LOOK HERE 👆👆👆 Need input 
```

Comment:
@HarrisonMS - what do you think about the file structure? Any input appreciated!